### DISCUSSION TITLE
Login page modifications

### DISCUSSION THREAD

**Discussion by tinpham5614** [SeattleColleges/belindas-closet-nextjs#49] [Show and tell]
Hi team,

I would like to discuss some changes to the login page on Belinda's website. I have made some modifications to the page, as shown in the attached screenshot. 
If my proposed changes are approved, I would like to request a new issue and push the updates. 
I kindly request that you review my modifications and provide your feedback.

Thank you.
**Before:**
<img src="https://github-production-user-asset-6210df.s3.amazonaws.com/72054441/256957852-8e42bae8-e0b0-4527-971a-635445d03985.png" width="40%" height="40%" >

**After:**
<img src="https://github-production-user-asset-6210df.s3.amazonaws.com/72054441/256958016-64dfc49a-8855-42dc-ac22-a1f89106833e.png" width="50%" height="50%" >

**Comment by brinkbrink** [comment]
I think that looks really good, Tin! Definitely an upgrade, aesthetically. The only thing I'd request is to check the contrast on the text to make sure it is accessible. [This link shows how to check the contrast with Chrome's Developer Tools](https://developers.google.com/codelabs/devtools-cvd#3) if you're not already familiar. You can also click through the entire CodeLab to understand the issue if you need more information. It might be fine but it also might not be. If you've already done this, that's awesome! Thank you!

**Reply by tinpham5614** [reply]
Thank you, Brink. I will look into that.

**Reply by tinpham5614** [reply]
This is really helpful information. I tested and slightly adjusted the contrast.
**This is the title text:**
<img src="https://github.com/SeattleColleges/belindas-closet-nextjs/assets/72054441/22e94b49-6c4a-4290-b400-3c4a6624b2c2.png" width="50%" height="50%">
**This is the button text:**
<img src="https://github-production-user-asset-6210df.s3.amazonaws.com/72054441/256960532-1f8ec16c-59fd-46aa-9ec5-9e5d542aa2c6.png" width="50%" height="50%">
